easybuild/easyconfigs/g/gdc-client/gdc-client-1.3.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,134 @@
+easyblock = 'Bundle'
```

Comment:
@fizwit We would like to see the new `PythonBundle` generic easyblock being adopted for easyconfig files like this, cfr. https://github.com/easybuilders/easybuild-easyblocks/pull/1553

I've made the necessary changes in https://github.com/fizwit/easybuild-easyconfigs/pull/5 for this PR.

Reply:
I am not able to update the PR because PythonBundle is not available in 3.7.1 yet.  I do not work with the Dev version of EB.

ERROR: Failed to process easyconfig /home/jfdey/git/easybuild-life-sciences/fh_easyconfigs/gdc-client-1.3.0-foss-2018b-Python-2.7.15.eb: Failed to obtain class for PythonBundle easyblock (not available?): No module named pythonbundle

Reply:
@fizwit OK, I understand, it makes sense to postpone requiring the use of `PythonBundle` until it's included in an EasyBuild release, indeed...